apiary_v2.apib

```diff
@@ -881,7 +884,11 @@ follows:
 ## Create an Order [POST]
 
 Once an order has been placed on the ecommerce platform, the Unmade system should be notified to begin the process of 
-generating manufacturing files.
+generating manufacturing files. To do this, pass a unique order reference, a shipping address and one or more order
+items to this endpoint via an HTTP POST request. Each item must reference an Unmade Design ID.
```

Comment:
👍🏽  Maybe we want to mention that they can "order" each design multiple times using the `options.size` ... I mean it's on the payload (not a biggy) just that the size needs to exist on our side 😄